src/test/scala/karazin/scala/users/group/week1/homework/HomeworkSpecification.scala

```diff
@@ -23,14 +24,30 @@ object BooleanOperatorsSpecification extends Properties("Boolean Operators"):
   property("and") = forAll { (pair: (Boolean, Boolean)) =>
     val (left, right) = pair
     
-    and(left, right) == left && right
+    and(left, right) == (left && right)
+  }
+
+  property("and with Exception") = forAll { (pair: (Boolean, Boolean)) =>

```

Comment:
Do you really need both `left` and `right` values?
You only need to check that when `left` is false then exception want be thrown.
Use the following construction instead of `forAll`:
```
  property("simple check") = propBoolean {
    1 == 2
  }
```